back/app/controllers/web_api/v1/projects_controller.rb

```diff
@@ -171,6 +171,19 @@ def votes_by_input_xlsx
     end
   end
 
+  def reset_participation_data
```

Comment:
Not the most crucial thing, but 
```suggestion
  def destroy_participation_data
```
would be more consistent with the Rails naming conventions.